docs/coconut.md

```diff
@@ -0,0 +1,112 @@
+# Coconut-SVSM
+
+## Installation
+
+For more details, please refer to [Coconut SVSM](https://github.com/coconut-svsm/svsm). The following instructions are intended to simplify the installation, especially for our usecase.
+### 1. Preparing the Host
+The host kernel needs support for SVSM.
+Install necessary dependencies:
+```bash
```

Comment:
Add newlines around code blocks (i.e. immediately above and below)

NB: please apply everywhere in this file